requirements.txt

```diff
@@ -90,49 +90,49 @@ docutils==0.19
     # via
     #   awscli
     #   galaxy-util
-ec2-metadata==2.16.0
+ec2-metadata==2.17.0
     # via toil
 enlighten==1.14.1
     # via toil
 executing==2.2.1
     # via stack-data
-filelock==3.19.1
+filelock==3.20.1
     # via cachecontrol
-fonttools==4.60.0
+fonttools==4.61.1
```

Comment:
https://github.com/advisories/GHSA-768j-98cg-p3fv 

Fix is in 4.61.2 which doesn't appear to be released yet https://pypi.org/project/fonttools/#history but there is no reason we can't pick up the upgrade when it's released as its constraints are merely:

```
  Collecting fonttools>=4.22.0 (from matplotlib->-r requirements.in (line 4))
    Obtaining dependency information for fonttools>=4.22.0 from https://files.pythonhosted.org/packages/4b/cf/00ba28b0990982530addb8dc3e9e6f2fa9cb5c20df2abdda7baa755e8fe1/fonttools-4.61.1-cp313-cp313-macosx_10_13_universal2.whl.metadata
    Using cached fonttools-4.61.1-cp313-cp313-macosx_10_13_universal2.whl.metadata (114 kB)
```

Commenting for transparency